lib/csdl2openapi.js

```diff
@@ -192,7 +192,9 @@ module.exports.csdl2openapi = function (
         Object.keys(terms).forEach(vocab => {
             voc[vocab] = {};
             terms[vocab].forEach(term => {
-                voc[vocab][term] = '@' + alias['Org.OData.' + vocab + '.V1'] + '.' + term;
+                const aliasKey = 'Org.OData.' + vocab + '.V1';
+                const prefix = alias[aliasKey] || aliasKey;
+                voc[vocab][term] = '@' + prefix + '.' + term;

```

Comment:
Not necessary, already covered by line 101

Reply:
I see the problem. We were are missing `<Reference>` declarations in the `MS Graph` [metadata](https://graph.microsoft.com/v1.0/$metadata). Not sure if the tool should error if theses are missing, since it causes unexpected behavioir? But it was a quick fix to just add them the the `MS Graph` export in any case.

---
without `<Reference>`s
`voc.Capabilities`

```json
{
   "BatchSupport":"@undefined.BatchSupport",
   "BatchSupported":"@undefined.BatchSupported",
   "ChangeTracking":"@undefined.ChangeTracking",
   "CountRestrictions":"@undefined.CountRestrictions",
   "DeleteRestrictions":"@undefined.DeleteRestrictions"
}
```

Reply:
Ah, I see! We can surely add that fallback, with a test case for it.